debian/control

```diff
@@ -10,7 +10,7 @@ Build-Depends: debhelper-compat (= 13),
                dh-exec,
                dh-golang,
                dctrl-tools,
-               golang-go (>= 2:1.22~),
+               golang-1.23-go,
```

Comment:
Should we go with an alternative? (I don’t know how this work btw when the package existing but doesn’t match the dep version). But something like `golang-go (>= 2:1.23~) | golang-1.23-go,` to be compatible both with future ubuntu releases and noble?

Reply:
Ah, I thought we wanted to support exactly one version as that's the one we test with, however changed the build-dependency and the order of the path, so that now the one installed in /usr/bin (if any) is preferred to the versioned one.

Reply:
I think Go is quite good at backward compatbility, so basically we want "at least" that version (which is the one we backported to previous releases)